docs/development-guide/IntegrationTests.md

```diff
@@ -69,3 +69,15 @@ cmd /c ./gradlew --no-parallel --no-daemon test-all-environments
 ```
 
 Test results can be found at `wrapper/build/report/index.html`.
+
+To re-run tests after making code changes, do a clean re-build so that the next test run will pick up the new changes:
```

Comment:
Appreciate the trouble shooting step here; however, `gradlew clean` is generally not needed after making code changes. It can be useful in situations when resolving unexpected build errors or unresolved references. Likewise, if there are major project structure changes (e.g., renaming packages), it can be useful to clean up any old artifacts. The general advice to run a clean build after every code change is not recommended.

```suggestion
If you encounter unexplained build issues or errors, try running the following to perform a clean build:
```